2023-07-28_16-43-46: DF Exception: Sub: R1191J, Sess: 1, ValueError, Events must correspond to the session with which the reader was initialized: 1 (events correspond to 0), file: create_events.py, line no: 587
2023-07-28_16-43-56: DF Exception: Sub: R1229M, Sess: 0, ValueError, Events must correspond to the session with which the reader was initialized: 0 (events correspond to 1), file: create_events.py, line no: 587
2023-07-28_16-44-05: DF Exception: Sub: R1269E, Sess: 3, ValueError, Events must correspond to the session with which the reader was initialized: 3 (events correspond to 10), file: create_events.py, line no: 587
2023-07-28_16-44-13: DF Exception: Sub: R1273D, Sess: 0, ValueError, Events must correspond to the session with which the reader was initialized: 0 (events correspond to 1), file: create_events.py, line no: 587
2023-07-28_16-44-21: DF Exception: Sub: R1273D, Sess: 1, ValueError, Events must correspond to the session with which the reader was initialized: 1 (events correspond to 2), file: create_events.py, line no: 587
2023-07-28_16-44-29: DF Exception: Sub: R1273D, Sess: 2, ValueError, Events must correspond to the session with which the reader was initialized: 2 (events correspond to 3), file: create_events.py, line no: 587
2023-07-28_18-00-43: DF Exception: Sub: R1426N, Sess: 1, ValueError, could not broadcast input array from shape (121,3600) into shape (121), file: create_events.py, line no: 587
2023-07-28_18-23-40: DF Exception: Sub: R1478T, Sess: 0, ValueError, len(channels) (188) must be the same length as the second data dimension (186), file: create_events.py, line no: 587
2023-07-28_18-23-51: DF Exception: Sub: R1478T, Sess: 1, ValueError, len(channels) (188) must be the same length as the second data dimension (186), file: create_events.py, line no: 587
2023-07-28_18-36-25: DF Exception: Sub: R1488T, Sess: 0, IndexError, index 91 is out of bounds for axis 0 with size 91, file: create_events.py, line no: 558
2023-07-28_19-32-39: DF Exception: Sub: R1626S, Sess: 0, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events.py, line no: 587
2023-07-28_19-32-50: DF Exception: Sub: R1626S, Sess: 2, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events.py, line no: 587
2023-07-28_19-33-02: DF Exception: Sub: R1626S, Sess: 3, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events.py, line no: 587
2023-07-28_19-33-14: DF Exception: Sub: R1626S, Sess: 4, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events.py, line no: 587
2023-07-28_19-33-27: DF Exception: Sub: R1626S, Sess: 6, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events.py, line no: 587
2023-07-28_19-33-39: DF Exception: Sub: R1626S, Sess: 7, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events.py, line no: 587
2023-07-28_19-33-49: DF Exception: Sub: R1626S, Sess: 8, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events.py, line no: 587
2023-07-28_19-41-22: DF Exception: Sub: R1221P, Sess: 2, ValueError, cannot reshape array of size 3599 into shape (2), file: create_events.py, line no: 745
2023-07-28_19-48-04: DF Exception: Sub: R1221P, Sess: 1, ValueError, cannot reshape array of size 3599 into shape (2), file: create_events.py, line no: 745
2023-07-28_19-51-10: DF Exception: Sub: R1221P, Sess: 0, ValueError, cannot reshape array of size 3599 into shape (2), file: create_events.py, line no: 745
2023-07-30_18-40-21: DF Exception: Sub: R1221P, Sess: 0, BdbQuit, , file: create_events.py, line no: 610
2023-07-31_13-24-33: DF Exception: Sub: R1221P, Sess: 0, BdbQuit, , file: create_events.py, line no: 610
2023-07-31_13-28-04: DF Exception: Sub: R1221P, Sess: 0, NameError, name 'x' is not defined, file: create_events_mne.py, line no: 376
2023-07-31_13-35-29: DF Exception: Sub: R1221P, Sess: 0, TypeError, compute_power_morlet() missing 1 required positional argument: 'eeg_buffer', file: create_events_mne.py, line no: 620
2023-07-31_13-38-48: DF Exception: Sub: R1221P, Sess: 0, TypeError, tfr_morlet() got an unexpected keyword argument 'njobs', file: create_events_mne.py, line no: 620
2023-07-31_13-48-42: DF Exception: Sub: R1221P, Sess: 0, ValueError, operands could not be broadcast together with shapes (10,300,131,30) (10,131) , file: create_events_mne.py, line no: 620
2023-07-31_14-11-58: DF Exception: Sub: R1221P, Sess: 0, IndexError, tuple index out of range, file: create_events_mne.py, line no: 621
2023-07-31_14-47-07: DF Exception: Sub: R1221P, Sess: 0, TypeError, apply_hilbert() got an unexpected keyword argument 'njobs', file: create_events_mne.py, line no: 648
2023-07-31_15-43-17: DF Exception: Sub: R1221P, Sess: 0, TypeError, apply_hilbert() got an unexpected keyword argument 'njobs', file: create_events_mne.py, line no: 647
2023-08-02_10-58-39: DF Exception: Sub: R1221P, Sess: 0, BdbQuit, , file: create_events_mne.py, line no: 602
2023-08-04_14-38-44: DF Exception: Sub: R1001P, Sess: 0, BdbQuit, , file: create_events_mne.py, line no: 308
2023-08-04_14-40-00: DF Exception: Sub: R1001P, Sess: 0, BdbQuit, , file: create_events_mne.py, line no: 308
2023-08-04_14-40-14: DF Exception: Sub: R1001P, Sess: 1, AttributeError, 'Pandas' object has no attribute 'sesion', file: create_events_mne.py, line no: 300
2023-08-04_14-40-35: DF Exception: Sub: R1001P, Sess: 1, AttributeError, 'Pandas' object has no attribute 'sesion', file: create_events_mne.py, line no: 300
2023-08-04_14-41-31: DF Exception: Sub: R1001P, Sess: 1, BdbQuit, , file: create_events_mne.py, line no: 308
2023-08-04_14-42-25: DF Exception: Sub: R1001P, Sess: 0, BdbQuit, , file: create_events_mne.py, line no: 308
2023-08-04_17-21-13: DF Exception: Sub: R1013E, Sess: 0, BdbQuit, , file: create_events_mne.py, line no: 270
2023-08-04_18-29-33: DF Exception: Sub: R1015J, Sess: 0, BdbQuit, , file: create_events_mne_2.py, line no: 684
2023-08-04_18-30-15: DF Exception: Sub: R1032D, Sess: 1, IndexError, index 11 is out of bounds for axis 1 with size 8, file: create_events_mne_2.py, line no: 683
2023-08-04_18-31-07: DF Exception: Sub: R1031M, Sess: 0, IndexError, index 2 is out of bounds for axis 1 with size 2, file: create_events_mne_2.py, line no: 683
2023-08-04_18-32-27: DF Exception: Sub: R1031M, Sess: 0, TypeError, only integer scalar arrays can be converted to a scalar index, file: create_events_mne_2.py, line no: 678
2023-08-04_18-50-31: DF Exception: Sub: R1032D, Sess: 0, ValueError, All picks must be < n_channels (8), got array([ 1,  2,  4, 11, 12, 13]), file: create_events_mne_2.py, line no: 857
2023-08-04_18-54-33: DF Exception: Sub: R1032D, Sess: 0, BdbQuit, , file: create_events_mne_2.py, line no: 649
2023-08-04_18-59-31: DF Exception: Sub: R1032D, Sess: 0, NameError, name 'ripple_logic' is not defined, file: create_events_mne_2.py, line no: 737
2023-08-04_19-08-45: DF Exception: Sub: R1026D, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-08-46: DF Exception: Sub: R1028M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-08-46: DF Exception: Sub: R1028M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-08-47: DF Exception: Sub: R1021D, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-08-49: DF Exception: Sub: R1016M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-08-49: DF Exception: Sub: R1016M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-08-53: DF Exception: Sub: R1013E, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-08-56: DF Exception: Sub: R1036M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-09-00: DF Exception: Sub: R1041M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-09-01: DF Exception: Sub: R1042M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-09-12: DF Exception: Sub: R1050M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-09-14: DF Exception: Sub: R1060M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-09-15: DF Exception: Sub: R1056M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=int64)), file: create_events_mne.py, line no: 648
2023-08-04_19-11-43: DF Exception: Sub: R1069M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-11-46: DF Exception: Sub: R1074M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-11-52: DF Exception: Sub: R1075J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-12-05: DF Exception: Sub: R1093J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-12-43: DF Exception: Sub: R1106M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=int64)), file: create_events_mne.py, line no: 648
2023-08-04_19-12-53: DF Exception: Sub: R1107J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-13-01: DF Exception: Sub: R1107J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-13-42: DF Exception: Sub: R1111M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-13-50: DF Exception: Sub: R1111M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-13-58: DF Exception: Sub: R1114C, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-14-09: DF Exception: Sub: R1119P, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-14-49: DF Exception: Sub: R1130M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-15-29: DF Exception: Sub: R1135E, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-15-44: DF Exception: Sub: R1135E, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-16-14: DF Exception: Sub: R1141T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-16-25: DF Exception: Sub: R1141T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-16-36: DF Exception: Sub: R1141T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-21-14: DF Exception: Sub: R1170J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-21-17: DF Exception: Sub: R1170J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-21-24: DF Exception: Sub: R1178P, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-22-51: DF Exception: Sub: R1181E, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-22-58: DF Exception: Sub: R1181E, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-23-06: DF Exception: Sub: R1181E, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-23-13: DF Exception: Sub: R1184M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=int64)), file: create_events_mne.py, line no: 648
2023-08-04_19-23-42: DF Exception: Sub: R1189M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-23-47: DF Exception: Sub: R1191J, Sess: 1, ValueError, Events must correspond to the session with which the reader was initialized: 1 (events correspond to 0), file: create_events_mne.py, line no: 594
2023-08-04_19-24-12: DF Exception: Sub: R1202M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-24-18: DF Exception: Sub: R1202M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-26-00: DF Exception: Sub: R1212P, Sess: 0, ValueError, cannot reshape array of size 4999 into shape (2), file: create_events_mne.py, line no: 697
2023-08-04_19-27-15: DF Exception: Sub: R1221P, Sess: 0, ValueError, cannot reshape array of size 4999 into shape (2), file: create_events_mne.py, line no: 697
2023-08-04_19-27-30: DF Exception: Sub: R1222M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-27-41: DF Exception: Sub: R1221P, Sess: 1, ValueError, cannot reshape array of size 4999 into shape (2), file: create_events_mne.py, line no: 697
2023-08-04_19-28-01: DF Exception: Sub: R1221P, Sess: 2, ValueError, cannot reshape array of size 4999 into shape (2), file: create_events_mne.py, line no: 697
2023-08-04_19-28-11: DF Exception: Sub: R1229M, Sess: 0, ValueError, Events must correspond to the session with which the reader was initialized: 0 (events correspond to 1), file: create_events_mne.py, line no: 594
2023-08-04_19-28-24: DF Exception: Sub: R1231M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-28-30: DF Exception: Sub: R1235E, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-28-32: DF Exception: Sub: R1235E, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-31-46: DF Exception: Sub: R1251M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-32-42: DF Exception: Sub: R1254E, Sess: 0, ValueError, cannot reshape array of size 4999 into shape (2), file: create_events_mne.py, line no: 697
2023-08-04_19-32-49: DF Exception: Sub: R1266J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-32-52: DF Exception: Sub: R1266J, Sess: 5, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-32-55: DF Exception: Sub: R1266J, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-32-56: DF Exception: Sub: R1266J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-33-06: DF Exception: Sub: R1269E, Sess: 3, ValueError, Events must correspond to the session with which the reader was initialized: 3 (events correspond to 10), file: create_events_mne.py, line no: 594
2023-08-04_19-33-09: DF Exception: Sub: R1273D, Sess: 0, ValueError, Events must correspond to the session with which the reader was initialized: 0 (events correspond to 1), file: create_events_mne.py, line no: 594
2023-08-04_19-33-14: DF Exception: Sub: R1273D, Sess: 2, ValueError, Events must correspond to the session with which the reader was initialized: 2 (events correspond to 3), file: create_events_mne.py, line no: 594
2023-08-04_19-33-24: DF Exception: Sub: R1274T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-33-24: DF Exception: Sub: R1274T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-33-28: DF Exception: Sub: R1273D, Sess: 1, ValueError, Events must correspond to the session with which the reader was initialized: 1 (events correspond to 2), file: create_events_mne.py, line no: 594
2023-08-04_19-33-33: DF Exception: Sub: R1274T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-33-40: DF Exception: Sub: R1277J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-33-43: DF Exception: Sub: R1277J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-34-39: DF Exception: Sub: R1290M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-35-14: DF Exception: Sub: R1302M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-35-18: DF Exception: Sub: R1302M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-36-04: DF Exception: Sub: R1317D, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-36-21: DF Exception: Sub: R1321M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-36-31: DF Exception: Sub: R1324M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-36-33: DF Exception: Sub: R1321M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-42-28: DF Exception: Sub: R1351M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-42-33: DF Exception: Sub: R1351M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-44-42: DF Exception: Sub: R1364C, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-45-02: DF Exception: Sub: R1364C, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-45-27: DF Exception: Sub: R1370E, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-46-28: DF Exception: Sub: R1375C, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-46-37: DF Exception: Sub: R1375C, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-46-37: DF Exception: Sub: R1375C, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-46-40: DF Exception: Sub: R1375C, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-47-01: DF Exception: Sub: R1378T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-47-09: DF Exception: Sub: R1378T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-49-32: DF Exception: Sub: R1384J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-49-41: DF Exception: Sub: R1384J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-50-32: DF Exception: Sub: R1389J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-50-33: DF Exception: Sub: R1389J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-50-41: DF Exception: Sub: R1390M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-50-42: DF Exception: Sub: R1390M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-50-42: DF Exception: Sub: R1390M, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-50-48: DF Exception: Sub: R1389J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-50-49: DF Exception: Sub: R1390M, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-50-49: DF Exception: Sub: R1389J, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-50-52: DF Exception: Sub: R1391T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-50-53: DF Exception: Sub: R1391T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-53-40: DF Exception: Sub: R1400N, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-53-44: DF Exception: Sub: R1400N, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-53-49: DF Exception: Sub: R1400N, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-53-53: DF Exception: Sub: R1400N, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-53-58: DF Exception: Sub: R1401J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-54-03: DF Exception: Sub: R1401J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-54-03: DF Exception: Sub: R1401J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-54-13: DF Exception: Sub: R1403N, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-54-16: DF Exception: Sub: R1403N, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-54-19: DF Exception: Sub: R1403N, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-54-21: DF Exception: Sub: R1403N, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-54-57: DF Exception: Sub: R1406M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-55-05: DF Exception: Sub: R1406M, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-55-12: DF Exception: Sub: R1406M, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-55-14: DF Exception: Sub: R1406M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-55-21: DF Exception: Sub: R1408N, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-55-32: DF Exception: Sub: R1408N, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-57-00: DF Exception: Sub: R1426N, Sess: 1, ValueError, could not broadcast input array from shape (121,5000) into shape (121), file: create_events_mne.py, line no: 594
2023-08-04_19-57-16: DF Exception: Sub: R1441T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_19-57-25: DF Exception: Sub: R1441T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-01-01: DF Exception: Sub: R1457T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-01-21: DF Exception: Sub: R1462M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-02-16: DF Exception: Sub: R1466J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-05-35: DF Exception: Sub: R1474T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-05-38: DF Exception: Sub: R1474T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-05-44: DF Exception: Sub: R1474T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-05-51: DF Exception: Sub: R1475J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-05-53: DF Exception: Sub: R1475J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-05-55: DF Exception: Sub: R1475J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-08-54: DF Exception: Sub: R1478T, Sess: 0, ValueError, len(channels) (188) must be the same length as the second data dimension (186), file: create_events_mne.py, line no: 594
2023-08-04_20-09-05: DF Exception: Sub: R1478T, Sess: 1, ValueError, len(channels) (188) must be the same length as the second data dimension (186), file: create_events_mne.py, line no: 594
2023-08-04_20-09-20: DF Exception: Sub: R1485J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-09-36: DF Exception: Sub: R1485J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-09-43: DF Exception: Sub: R1485J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-10-04: DF Exception: Sub: R1485J, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-10-55: DF Exception: Sub: R1488T, Sess: 0, IndexError, index 91 is out of bounds for axis 0 with size 91, file: create_events_mne.py, line no: 566
2023-08-04_20-11-57: DF Exception: Sub: R1490T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-12-06: DF Exception: Sub: R1490T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-12-16: DF Exception: Sub: R1490T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-13-22: DF Exception: Sub: R1492J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-13-45: DF Exception: Sub: R1492J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-13-53: DF Exception: Sub: R1492J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-13-57: DF Exception: Sub: R1492J, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-14-09: DF Exception: Sub: R1495J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-14-10: DF Exception: Sub: R1495J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-14-12: DF Exception: Sub: R1495J, Sess: 10, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-14-21: DF Exception: Sub: R1495J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-14-21: DF Exception: Sub: R1495J, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-14-22: DF Exception: Sub: R1495J, Sess: 5, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-14-32: DF Exception: Sub: R1495J, Sess: 8, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-14-32: DF Exception: Sub: R1495J, Sess: 6, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-16-13: DF Exception: Sub: R1500T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-16-19: DF Exception: Sub: R1500T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-16-25: DF Exception: Sub: R1500T, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-16-41: DF Exception: Sub: R1500T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-19-38: DF Exception: Sub: R1526J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-27-29: DF Exception: Sub: R1626S, Sess: 0, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 594
2023-08-04_20-27-32: DF Exception: Sub: R1626S, Sess: 2, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 594
2023-08-04_20-27-40: DF Exception: Sub: R1626S, Sess: 4, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 594
2023-08-04_20-27-43: DF Exception: Sub: R1626S, Sess: 6, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 594
2023-08-04_20-27-47: DF Exception: Sub: R1626S, Sess: 3, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 594
2023-08-04_20-27-51: DF Exception: Sub: R1626S, Sess: 7, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 594
2023-08-04_20-27-53: DF Exception: Sub: R1626S, Sess: 8, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 594
2023-08-04_20-41-40: DF Exception: Sub: R1681J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-04_20-41-55: DF Exception: Sub: R1681J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 648
2023-08-06_20-22-20: DF Exception: Sub: R1689J, Sess: 0, BdbQuit, , file: create_events_mne.py, line no: 681
2023-08-06_20-35-29: DF Exception: Sub: R1689J, Sess: 0, BdbQuit, , file: create_events_mne.py, line no: 623
2023-08-06_21-09-01: DF Exception: Sub: R1689J, Sess: 0, BdbQuit, , file: create_events_mne.py, line no: 630
2023-08-06_21-12-13: DF Exception: Sub: R1013E, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 646
2023-08-06_21-12-43: DF Exception: Sub: R1016M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 646
2023-08-06_21-13-03: DF Exception: Sub: R1016M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 646
2023-08-06_21-13-23: DF Exception: Sub: R1021D, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 646
2023-08-06_21-14-32: DF Exception: Sub: R1024E, Sess: 0, IndexError, list index out of range, file: create_events_mne.py, line no: 821
2023-08-06_21-21-35: DF Exception: Sub: R1026D, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-22-13: DF Exception: Sub: R1028M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-22-29: DF Exception: Sub: R1028M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-23-09: DF Exception: Sub: R1036M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-23-37: DF Exception: Sub: R1041M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-23-52: DF Exception: Sub: R1042M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-30-58: DF Exception: Sub: R1060M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-31-00: DF Exception: Sub: R1056M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=int64)), file: create_events_mne.py, line no: 647
2023-08-06_21-31-00: DF Exception: Sub: R1050M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-39-55: DF Exception: Sub: R1069M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-40-04: DF Exception: Sub: R1074M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-40-10: DF Exception: Sub: R1075J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-41-07: DF Exception: Sub: R1093J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-42-30: DF Exception: Sub: R1106M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=int64)), file: create_events_mne.py, line no: 647
2023-08-06_21-42-43: DF Exception: Sub: R1107J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-42-52: DF Exception: Sub: R1107J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-45-00: DF Exception: Sub: R1111M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-45-08: DF Exception: Sub: R1111M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-46-59: DF Exception: Sub: R1114C, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-47-08: DF Exception: Sub: R1119P, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-49-55: DF Exception: Sub: R1130M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-50-30: DF Exception: Sub: R1135E, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-50-53: DF Exception: Sub: R1135E, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-54-56: DF Exception: Sub: R1141T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-54-57: DF Exception: Sub: R1141T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_21-55-03: DF Exception: Sub: R1141T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-06-00: DF Exception: Sub: R1170J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-06-07: DF Exception: Sub: R1170J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-07-48: DF Exception: Sub: R1178P, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-09-17: DF Exception: Sub: R1181E, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-09-25: DF Exception: Sub: R1181E, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-09-31: DF Exception: Sub: R1184M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=int64)), file: create_events_mne.py, line no: 647
2023-08-06_22-09-31: DF Exception: Sub: R1181E, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-09-59: DF Exception: Sub: R1189M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-11-33: DF Exception: Sub: R1191J, Sess: 1, ValueError, Events must correspond to the session with which the reader was initialized: 1 (events correspond to 0), file: create_events_mne.py, line no: 594
2023-08-06_22-12-12: DF Exception: Sub: R1202M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-12-24: DF Exception: Sub: R1202M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-14-43: DF Exception: Sub: R1212P, Sess: 0, ValueError, cannot reshape array of size 4999 into shape (2), file: create_events_mne.py, line no: 699
2023-08-06_22-15-25: DF Exception: Sub: R1221P, Sess: 1, ValueError, cannot reshape array of size 4999 into shape (2), file: create_events_mne.py, line no: 699
2023-08-06_22-15-33: DF Exception: Sub: R1222M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-15-46: DF Exception: Sub: R1221P, Sess: 2, ValueError, cannot reshape array of size 4999 into shape (2), file: create_events_mne.py, line no: 699
2023-08-06_22-15-52: DF Exception: Sub: R1221P, Sess: 0, ValueError, cannot reshape array of size 4999 into shape (2), file: create_events_mne.py, line no: 699
2023-08-06_22-17-13: DF Exception: Sub: R1229M, Sess: 0, ValueError, Events must correspond to the session with which the reader was initialized: 0 (events correspond to 1), file: create_events_mne.py, line no: 594
2023-08-06_22-17-53: DF Exception: Sub: R1231M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-18-00: DF Exception: Sub: R1235E, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-18-02: DF Exception: Sub: R1235E, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-22-17: DF Exception: Sub: R1251M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-22-48: DF Exception: Sub: R1254E, Sess: 0, ValueError, cannot reshape array of size 4999 into shape (2), file: create_events_mne.py, line no: 699
2023-08-06_22-23-00: DF Exception: Sub: R1266J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-23-15: DF Exception: Sub: R1266J, Sess: 5, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-23-17: DF Exception: Sub: R1266J, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-23-17: DF Exception: Sub: R1266J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-23-27: DF Exception: Sub: R1269E, Sess: 3, ValueError, Events must correspond to the session with which the reader was initialized: 3 (events correspond to 10), file: create_events_mne.py, line no: 594
2023-08-06_22-23-38: DF Exception: Sub: R1273D, Sess: 0, ValueError, Events must correspond to the session with which the reader was initialized: 0 (events correspond to 1), file: create_events_mne.py, line no: 594
2023-08-06_22-23-46: DF Exception: Sub: R1273D, Sess: 1, ValueError, Events must correspond to the session with which the reader was initialized: 1 (events correspond to 2), file: create_events_mne.py, line no: 594
2023-08-06_22-23-54: DF Exception: Sub: R1273D, Sess: 2, ValueError, Events must correspond to the session with which the reader was initialized: 2 (events correspond to 3), file: create_events_mne.py, line no: 594
2023-08-06_22-23-54: DF Exception: Sub: R1274T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-24-05: DF Exception: Sub: R1274T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-24-06: DF Exception: Sub: R1274T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-24-21: DF Exception: Sub: R1277J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-24-23: DF Exception: Sub: R1277J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-26-16: DF Exception: Sub: R1290M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-26-29: DF Exception: Sub: R1302M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-26-52: DF Exception: Sub: R1302M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-29-37: DF Exception: Sub: R1317D, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-32-15: DF Exception: Sub: R1321M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-32-17: DF Exception: Sub: R1324M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-32-23: DF Exception: Sub: R1321M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-47-45: DF Exception: Sub: R1351M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-47-54: DF Exception: Sub: R1351M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-51-35: DF Exception: Sub: R1364C, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-51-52: DF Exception: Sub: R1364C, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-53-52: DF Exception: Sub: R1370E, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-56-26: DF Exception: Sub: R1375C, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-56-51: DF Exception: Sub: R1375C, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-56-56: DF Exception: Sub: R1375C, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-56-57: DF Exception: Sub: R1375C, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-57-46: DF Exception: Sub: R1378T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_22-57-58: DF Exception: Sub: R1378T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-01-49: DF Exception: Sub: R1384J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-01-55: DF Exception: Sub: R1384J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-03-26: DF Exception: Sub: R1389J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-03-30: DF Exception: Sub: R1389J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-03-36: DF Exception: Sub: R1389J, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-03-41: DF Exception: Sub: R1389J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-03-43: DF Exception: Sub: R1390M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-03-46: DF Exception: Sub: R1390M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-03-52: DF Exception: Sub: R1390M, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-03-53: DF Exception: Sub: R1390M, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-03-56: DF Exception: Sub: R1391T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-03-56: DF Exception: Sub: R1391T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-09-03: DF Exception: Sub: R1400N, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-09-10: DF Exception: Sub: R1400N, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-09-16: DF Exception: Sub: R1401J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-09-18: DF Exception: Sub: R1400N, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-09-19: DF Exception: Sub: R1401J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-09-19: DF Exception: Sub: R1400N, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-09-28: DF Exception: Sub: R1403N, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-09-29: DF Exception: Sub: R1403N, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-09-30: DF Exception: Sub: R1401J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-09-33: DF Exception: Sub: R1403N, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-09-34: DF Exception: Sub: R1403N, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-09-50: DF Exception: Sub: R1406M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-10-01: DF Exception: Sub: R1406M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-10-07: DF Exception: Sub: R1406M, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-10-07: DF Exception: Sub: R1406M, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-10-13: DF Exception: Sub: R1408N, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-10-19: DF Exception: Sub: R1408N, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-13-48: DF Exception: Sub: R1426N, Sess: 1, ValueError, could not broadcast input array from shape (121,5000) into shape (121), file: create_events_mne.py, line no: 594
2023-08-06_23-15-02: DF Exception: Sub: R1441T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-15-16: DF Exception: Sub: R1441T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-20-45: DF Exception: Sub: R1457T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-21-29: DF Exception: Sub: R1462M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-23-13: DF Exception: Sub: R1466J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-30-17: DF Exception: Sub: R1474T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-30-19: DF Exception: Sub: R1474T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-30-23: DF Exception: Sub: R1474T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-30-36: DF Exception: Sub: R1475J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-30-36: DF Exception: Sub: R1475J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-30-37: DF Exception: Sub: R1475J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-32-16: DF Exception: Sub: R1478T, Sess: 0, ValueError, len(channels) (188) must be the same length as the second data dimension (186), file: create_events_mne.py, line no: 594
2023-08-06_23-32-17: DF Exception: Sub: R1478T, Sess: 1, ValueError, len(channels) (188) must be the same length as the second data dimension (186), file: create_events_mne.py, line no: 594
2023-08-06_23-35-44: DF Exception: Sub: R1485J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-35-55: DF Exception: Sub: R1485J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-36-11: DF Exception: Sub: R1485J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-36-20: DF Exception: Sub: R1485J, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-38-14: DF Exception: Sub: R1488T, Sess: 0, IndexError, index 91 is out of bounds for axis 0 with size 91, file: create_events_mne.py, line no: 566
2023-08-06_23-39-47: DF Exception: Sub: R1490T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-39-52: DF Exception: Sub: R1490T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-40-00: DF Exception: Sub: R1490T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-41-32: DF Exception: Sub: R1492J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-41-48: DF Exception: Sub: R1492J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-42-04: DF Exception: Sub: R1492J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-42-23: DF Exception: Sub: R1492J, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-44-04: DF Exception: Sub: R1495J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-44-04: DF Exception: Sub: R1495J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-44-15: DF Exception: Sub: R1495J, Sess: 10, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-44-20: DF Exception: Sub: R1495J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-44-25: DF Exception: Sub: R1495J, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-44-35: DF Exception: Sub: R1495J, Sess: 5, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-44-43: DF Exception: Sub: R1495J, Sess: 8, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-44-43: DF Exception: Sub: R1495J, Sess: 6, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-47-51: DF Exception: Sub: R1500T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-47-54: DF Exception: Sub: R1500T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-48-00: DF Exception: Sub: R1500T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-48-08: DF Exception: Sub: R1500T, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-06_23-54-38: DF Exception: Sub: R1526J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-07_00-09-48: DF Exception: Sub: R1626S, Sess: 2, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 594
2023-08-07_00-09-52: DF Exception: Sub: R1626S, Sess: 0, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 594
2023-08-07_00-10-03: DF Exception: Sub: R1626S, Sess: 3, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 594
2023-08-07_00-10-13: DF Exception: Sub: R1626S, Sess: 4, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 594
2023-08-07_00-10-18: DF Exception: Sub: R1626S, Sess: 7, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 594
2023-08-07_00-10-19: DF Exception: Sub: R1626S, Sess: 6, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 594
2023-08-07_00-10-39: DF Exception: Sub: R1626S, Sess: 8, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 594
2023-08-07_00-36-43: DF Exception: Sub: R1681J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-07_00-36-58: DF Exception: Sub: R1681J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 647
2023-08-12_19-15-30: DF Exception: Sub: R1016M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-15-31: DF Exception: Sub: R1026D, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-15-34: DF Exception: Sub: R1016M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-15-35: DF Exception: Sub: R1021D, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-15-36: DF Exception: Sub: R1004D, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-15-42: DF Exception: Sub: R1035M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-15-43: DF Exception: Sub: R1041M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-15-47: DF Exception: Sub: R1042M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-15-49: DF Exception: Sub: R1036M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-15-49: DF Exception: Sub: R1013E, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-15-54: DF Exception: Sub: R1050M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-15-58: DF Exception: Sub: R1060M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-16-05: DF Exception: Sub: R1061T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-16-13: DF Exception: Sub: R1065J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-16-21: DF Exception: Sub: R1065J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-16-27: DF Exception: Sub: R1065J, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-16-28: DF Exception: Sub: R1065J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-16-31: DF Exception: Sub: R1066P, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-16-33: DF Exception: Sub: R1066P, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-16-40: DF Exception: Sub: R1066P, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-16-52: DF Exception: Sub: R1074M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-16-54: DF Exception: Sub: R1069M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-17-00: DF Exception: Sub: R1086M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-17-06: DF Exception: Sub: R1075J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-17-11: DF Exception: Sub: R1092J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-17-13: DF Exception: Sub: R1093J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-17-22: DF Exception: Sub: R1107J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-17-26: DF Exception: Sub: R1107J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-18-35: DF Exception: Sub: R1111M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-18-35: DF Exception: Sub: R1111M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-19-06: DF Exception: Sub: R1127P, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-19-09: DF Exception: Sub: R1130M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-19-22: DF Exception: Sub: R1135E, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-19-27: DF Exception: Sub: R1138T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-19-27: DF Exception: Sub: R1135E, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-19-28: DF Exception: Sub: R1138T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-19-33: DF Exception: Sub: R1141T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-19-34: DF Exception: Sub: R1141T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-19-40: DF Exception: Sub: R1141T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-19-55: DF Exception: Sub: R1147P, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-20-30: DF Exception: Sub: R1167M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-20-33: DF Exception: Sub: R1169P, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-20-38: DF Exception: Sub: R1170J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-20-41: DF Exception: Sub: R1170J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-20-49: DF Exception: Sub: R1178P, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-21-16: DF Exception: Sub: R1181E, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-21-23: DF Exception: Sub: R1181E, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-21-25: DF Exception: Sub: R1181E, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-21-28: DF Exception: Sub: R1184M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-21-34: DF Exception: Sub: R1187P, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-21-35: DF Exception: Sub: R1189M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-21-38: DF Exception: Sub: R1188C, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-21-41: DF Exception: Sub: R1191J, Sess: 1, ValueError, Events must correspond to the session with which the reader was initialized: 1 (events correspond to 0), file: create_events_mne.py, line no: 587
2023-08-12_19-21-41: DF Exception: Sub: R1192C, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-21-47: DF Exception: Sub: R1192C, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-21-50: DF Exception: Sub: R1202M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-21-50: DF Exception: Sub: R1192C, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-21-52: DF Exception: Sub: R1202M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-21-52: DF Exception: Sub: R1201P, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-21-58: DF Exception: Sub: R1204T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-22-17: DF Exception: Sub: R1212P, Sess: 0, ValueError, cannot reshape array of size 4999 into shape (2), file: create_events_mne.py, line no: 692
2023-08-12_19-22-29: DF Exception: Sub: R1217T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-22-37: DF Exception: Sub: R1222M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-22-41: DF Exception: Sub: R1221P, Sess: 0, ValueError, cannot reshape array of size 4999 into shape (2), file: create_events_mne.py, line no: 692
2023-08-12_19-22-45: DF Exception: Sub: R1226D, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-22-48: DF Exception: Sub: R1226D, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-22-51: DF Exception: Sub: R1226D, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-22-52: DF Exception: Sub: R1226D, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-22-54: DF Exception: Sub: R1227T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-22-56: DF Exception: Sub: R1229M, Sess: 0, ValueError, Events must correspond to the session with which the reader was initialized: 0 (events correspond to 1), file: create_events_mne.py, line no: 587
2023-08-12_19-22-59: DF Exception: Sub: R1221P, Sess: 1, ValueError, cannot reshape array of size 4999 into shape (2), file: create_events_mne.py, line no: 692
2023-08-12_19-23-03: DF Exception: Sub: R1235E, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-23-03: DF Exception: Sub: R1230J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-23-04: DF Exception: Sub: R1221P, Sess: 2, ValueError, cannot reshape array of size 4999 into shape (2), file: create_events_mne.py, line no: 692
2023-08-12_19-23-06: DF Exception: Sub: R1231M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-23-10: DF Exception: Sub: R1235E, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-23-44: DF Exception: Sub: R1245E, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-23-52: DF Exception: Sub: R1251M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-24-19: DF Exception: Sub: R1264P, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-24-25: DF Exception: Sub: R1264P, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-24-28: DF Exception: Sub: R1254E, Sess: 0, ValueError, cannot reshape array of size 4999 into shape (2), file: create_events_mne.py, line no: 692
2023-08-12_19-24-55: DF Exception: Sub: R1269E, Sess: 3, ValueError, Events must correspond to the session with which the reader was initialized: 3 (events correspond to 10), file: create_events_mne.py, line no: 587
2023-08-12_19-25-00: DF Exception: Sub: R1271P, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-03: DF Exception: Sub: R1273D, Sess: 0, ValueError, Events must correspond to the session with which the reader was initialized: 0 (events correspond to 1), file: create_events_mne.py, line no: 587
2023-08-12_19-25-03: DF Exception: Sub: R1271P, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-05: DF Exception: Sub: R1273D, Sess: 1, ValueError, Events must correspond to the session with which the reader was initialized: 1 (events correspond to 2), file: create_events_mne.py, line no: 587
2023-08-12_19-25-05: DF Exception: Sub: R1271P, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-06: DF Exception: Sub: R1273D, Sess: 2, ValueError, Events must correspond to the session with which the reader was initialized: 2 (events correspond to 3), file: create_events_mne.py, line no: 587
2023-08-12_19-25-09: DF Exception: Sub: R1274T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-10: DF Exception: Sub: R1274T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-12: DF Exception: Sub: R1274T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-13: DF Exception: Sub: R1275D, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-17: DF Exception: Sub: R1277J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-19: DF Exception: Sub: R1277J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-36: DF Exception: Sub: R1291M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-38: DF Exception: Sub: R1290M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-41: DF Exception: Sub: R1291M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-44: DF Exception: Sub: R1302M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-47: DF Exception: Sub: R1293P, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-48: DF Exception: Sub: R1302M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-58: DF Exception: Sub: R1317D, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-25-58: DF Exception: Sub: R1313J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-26-17: DF Exception: Sub: R1321M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-26-17: DF Exception: Sub: R1321M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-26-24: DF Exception: Sub: R1324M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-27-09: DF Exception: Sub: R1334T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-27-46: DF Exception: Sub: R1338T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-28-43: DF Exception: Sub: R1351M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-28-54: DF Exception: Sub: R1351M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-30-22: DF Exception: Sub: R1364C, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-30-34: DF Exception: Sub: R1364C, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-30-50: DF Exception: Sub: R1368T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-30-59: DF Exception: Sub: R1372C, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-31-25: DF Exception: Sub: R1372C, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-31-31: DF Exception: Sub: R1375C, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-31-32: DF Exception: Sub: R1372C, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-31-36: DF Exception: Sub: R1375C, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-31-41: DF Exception: Sub: R1375C, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-31-53: DF Exception: Sub: R1375C, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-32-00: DF Exception: Sub: R1378T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-32-07: DF Exception: Sub: R1378T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-32-59: DF Exception: Sub: R1384J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-33-06: DF Exception: Sub: R1384J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-33-29: DF Exception: Sub: R1386T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-33-34: DF Exception: Sub: R1388T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-33-36: DF Exception: Sub: R1388T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-33-40: DF Exception: Sub: R1389J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-33-42: DF Exception: Sub: R1389J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-33-46: DF Exception: Sub: R1389J, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-33-47: DF Exception: Sub: R1389J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-33-52: DF Exception: Sub: R1390M, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-33-52: DF Exception: Sub: R1390M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-33-54: DF Exception: Sub: R1390M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-33-57: DF Exception: Sub: R1390M, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-34-09: DF Exception: Sub: R1393T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-34-09: DF Exception: Sub: R1393T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-34-17: DF Exception: Sub: R1393T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-35-29: DF Exception: Sub: R1396T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-35-59: DF Exception: Sub: R1400N, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-00: DF Exception: Sub: R1400N, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-02: DF Exception: Sub: R1400N, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-08: DF Exception: Sub: R1400N, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-09: DF Exception: Sub: R1403N, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-10: DF Exception: Sub: R1401J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-11: DF Exception: Sub: R1401J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-11: DF Exception: Sub: R1401J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-14: DF Exception: Sub: R1403N, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-14: DF Exception: Sub: R1404E, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-14: DF Exception: Sub: R1403N, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-17: DF Exception: Sub: R1403N, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-18: DF Exception: Sub: R1404E, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-20: DF Exception: Sub: R1406M, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-20: DF Exception: Sub: R1406M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-23: DF Exception: Sub: R1406M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-27: DF Exception: Sub: R1408N, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-28: DF Exception: Sub: R1406M, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-29: DF Exception: Sub: R1408N, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-33: DF Exception: Sub: R1412M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-34: DF Exception: Sub: R1412M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-40: DF Exception: Sub: R1412M, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-48: DF Exception: Sub: R1421M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-54: DF Exception: Sub: R1421M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-36-59: DF Exception: Sub: R1422T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-37-04: DF Exception: Sub: R1426N, Sess: 1, ValueError, could not broadcast input array from shape (121,5000) into shape (121), file: create_events_mne.py, line no: 587
2023-08-12_19-37-14: DF Exception: Sub: R1433E, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-37-17: DF Exception: Sub: R1427T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-37-22: DF Exception: Sub: R1441T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-37-25: DF Exception: Sub: R1441T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-38-15: DF Exception: Sub: R1445E, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-39-42: DF Exception: Sub: R1457T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-39-52: DF Exception: Sub: R1459M, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-40-08: DF Exception: Sub: R1462M, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-40-59: DF Exception: Sub: R1466J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-42-40: DF Exception: Sub: R1472T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-42-40: DF Exception: Sub: R1472T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-43-11: DF Exception: Sub: R1475J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-43-24: DF Exception: Sub: R1475J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-43-35: DF Exception: Sub: R1475J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-44-25: DF Exception: Sub: R1478T, Sess: 1, ValueError, len(channels) (188) must be the same length as the second data dimension (186), file: create_events_mne.py, line no: 587
2023-08-12_19-44-29: DF Exception: Sub: R1478T, Sess: 0, ValueError, len(channels) (188) must be the same length as the second data dimension (186), file: create_events_mne.py, line no: 587
2023-08-12_19-44-38: DF Exception: Sub: R1484T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-44-38: DF Exception: Sub: R1485J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-44-40: DF Exception: Sub: R1484T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-44-41: DF Exception: Sub: R1485J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-44-47: DF Exception: Sub: R1485J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-44-48: DF Exception: Sub: R1485J, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-44-51: DF Exception: Sub: R1486J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-44-52: DF Exception: Sub: R1486J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-44-53: DF Exception: Sub: R1488T, Sess: 0, IndexError, index 91 is out of bounds for axis 0 with size 91, file: create_events_mne.py, line no: 559
2023-08-12_19-44-54: DF Exception: Sub: R1486J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-44-56: DF Exception: Sub: R1486J, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-45-18: DF Exception: Sub: R1490T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-45-25: DF Exception: Sub: R1490T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-45-26: DF Exception: Sub: R1490T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-45-33: DF Exception: Sub: R1492J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-45-41: DF Exception: Sub: R1492J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-45-43: DF Exception: Sub: R1492J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-45-48: DF Exception: Sub: R1492J, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-45-50: DF Exception: Sub: R1493T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-45-58: DF Exception: Sub: R1493T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-45-58: DF Exception: Sub: R1493T, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-46-06: DF Exception: Sub: R1495J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-46-06: DF Exception: Sub: R1493T, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-46-16: DF Exception: Sub: R1495J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-46-18: DF Exception: Sub: R1495J, Sess: 10, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-46-23: DF Exception: Sub: R1495J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-46-30: DF Exception: Sub: R1495J, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-46-30: DF Exception: Sub: R1495J, Sess: 5, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-46-37: DF Exception: Sub: R1495J, Sess: 8, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-46-41: DF Exception: Sub: R1495J, Sess: 6, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-47-10: DF Exception: Sub: R1497T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-47-25: DF Exception: Sub: R1497T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-47-34: DF Exception: Sub: R1497T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-47-38: DF Exception: Sub: R1497T, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-47-39: DF Exception: Sub: R1497T, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-47-58: DF Exception: Sub: R1500T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-47-59: DF Exception: Sub: R1500T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-47-59: DF Exception: Sub: R1500T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-48-06: DF Exception: Sub: R1500T, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-49-39: DF Exception: Sub: R1526J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-50-21: DF Exception: Sub: R1530J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-50-29: DF Exception: Sub: R1530J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-50-31: DF Exception: Sub: R1530J, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-50-31: DF Exception: Sub: R1530J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-50-34: DF Exception: Sub: R1530J, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-50-36: DF Exception: Sub: R1530J, Sess: 6, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-50-42: DF Exception: Sub: R1532T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-50-43: DF Exception: Sub: R1532T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-50-51: DF Exception: Sub: R1537T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-51-14: DF Exception: Sub: R1542J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-51-20: DF Exception: Sub: R1542J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-52-11: DF Exception: Sub: R1626S, Sess: 0, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 587
2023-08-12_19-52-20: DF Exception: Sub: R1626S, Sess: 3, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 587
2023-08-12_19-52-21: DF Exception: Sub: R1626S, Sess: 2, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 587
2023-08-12_19-52-23: DF Exception: Sub: R1626S, Sess: 4, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 587
2023-08-12_19-52-24: DF Exception: Sub: R1626S, Sess: 6, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 587
2023-08-12_19-52-29: DF Exception: Sub: R1626S, Sess: 8, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 587
2023-08-12_19-52-29: DF Exception: Sub: R1626S, Sess: 7, ValueError, len(channels) (189) must be the same length as the second data dimension (156), file: create_events_mne.py, line no: 587
2023-08-12_19-52-43: DF Exception: Sub: R1637T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-55-25: DF Exception: Sub: R1659A, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-55-30: DF Exception: Sub: R1659A, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-55-35: DF Exception: Sub: R1659A, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-55-36: DF Exception: Sub: R1659A, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-55-37: DF Exception: Sub: R1662S, Sess: 10, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-55-41: DF Exception: Sub: R1662S, Sess: 5, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-55-47: DF Exception: Sub: R1662S, Sess: 6, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-55-48: DF Exception: Sub: R1662S, Sess: 7, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-55-49: DF Exception: Sub: R1662S, Sess: 8, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-55-52: DF Exception: Sub: R1662S, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-55-58: DF Exception: Sub: R1666S, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-56-03: DF Exception: Sub: R1666S, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-56-07: DF Exception: Sub: R1666S, Sess: 5, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-56-09: DF Exception: Sub: R1666S, Sess: 4, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-56-14: DF Exception: Sub: R1666S, Sess: 6, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-56-18: DF Exception: Sub: R1666S, Sess: 7, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-56-29: DF Exception: Sub: R1670J, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-56-32: DF Exception: Sub: R1670J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-56-33: DF Exception: Sub: R1670J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-56-38: DF Exception: Sub: R1670J, Sess: 3, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-57-23: DF Exception: Sub: R1678T, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-57-26: DF Exception: Sub: R1678T, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-57-31: DF Exception: Sub: R1678T, Sess: 2, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-57-59: DF Exception: Sub: R1681J, Sess: 0, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
2023-08-12_19-58-09: DF Exception: Sub: R1681J, Sess: 1, ValueError, No appropriate channels found for the given picks (array([], dtype=float64)), file: create_events_mne.py, line no: 640
